e2e/e2e_test.go

```diff
@@ -332,5 +348,90 @@ var _ = Describe("Mock test", func() {
 				Eventually(tapeSession.Out).Should(Say("Time.*Block.*Tx.*10.*"))
 			})
 		})
+
+		When("Multiple committers", func() {
```

Comment:
Please write more descriptive text for test cases - they should describe what actually is being done during test